notebooks/290-stable-diffusion-keras-cv/290-stable-diffusion-keras-cv.ipynb

```diff
@@ -0,0 +1,722 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        "photograph of an astronaut riding a horse",
</pre><p>generated image quality not looks great.. also I do not see seed in this line, it means that user will have different result from run to run here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Seed added</p>